json11.cpp

```diff
@@ -375,38 +375,38 @@ struct JsonParser final {
       if (str[i] == '/') {
         i++;
         if (i == str.size())
-          return fail("unexpected end of input inside comment", 0);
+          return !!fail("unexpected end of input inside comment", 0);
```

Comment:
Ah right. I didn't really investigate the `fail` function aside from its interface. I'll make the change!